### ISSUE TITLE
[Caffe2] Retraining saved model

### ISSUE THREAD

**Issue by pjavia-iterativescopes** [pytorch/pytorch#19244]
https://github.com/caffe2/tutorials/blob/master/CIFAR10_Part2.ipynb

Above documentation is incorrect. Also, the Exporter fails if a model has Batch Norm in it. Also, following the same steps in above document for retraining a model gives a segmentation fault.

Please guide. I also posted this on PyTorch discuss but there is no response.
At least, a good working document to load model(which has Batch Norm) and retrain it would be really helpful.

**Comment by pjavia-iterativescopes** [comment]
I figured it out. For inference with a model having Batch Normalization, if you are saving using method Export, please add *_rm and *_riv as extra blobs and then call the Export method. However, if you want to retrain it then don't add *_rm and *_riv, in that case just use the current Export method.

I had one quick question though in the Export definition in caffe2.python.predictor.mobile_exporter what does blob version mean?
